### ISSUE TITLE
Add a few example layouts to start with?

### ISSUE THREAD

**Issue by niksmac** [uber/baseweb#3885]
Adding a few example layouts to start with would be very useful for people coming from other systems. 

## Feature description

Some examples like
- How to make a homepage with a few sections
- A sign up form
- May be a blog layout. 

Personally I end up having a 8px margin around my body tag, struggling to get out of it.

**Comment by qiwang97** [comment]
Must be useful for i change theme failure at the first time and some errors occur.

**Comment by sandgraham** [comment]
> Must be useful for i change theme failure at the first time and some errors occur.

@qiwang97 - could you be a tad more specific? If you need help with the theme there is a rather in detail guide [here](https://baseweb.design/guides/theming/).

**Comment by jkhaui** [comment]
I actually really like baseweb; far more than material-ui. However, material-ui has such an established community around it with entire apps, themes, and templates (free and paid) built using it as their core.

More docs and examples are nice, but what is really needed imo is to grow a community by providing entire sample apps to work off.

**Comment by qiwang97** [comment]
> > Must be useful for i change theme failure at the first time and some errors occur.
> 
> @qiwang97 could you be a tad more specific? If you need help with the theme there is a rather in detail guide [here](https://baseweb.design/guides/theming/).

Don't mind.  I mean what @jkhaui said.

**Comment by niksmac** [comment]
@jkhaui the one thing I liked with baseweb is the bundle size, my app is only a few Kbs in production and I love it.

**Comment by jkhaui** [comment]
> @jkhaui the one thing I liked with baseweb is the bundle size, my app is only a few Kbs in production and I love it.

Yeah, the CSS class names generation by styletron is fantastic. I much prefer it to material-ui's system which seems far too complex.

I've been building a couple of frontend apps with baseweb. Hopefully I can modify them to be generic and reusable (e.g. a dashboard). Then I'll host them somewhere as starter templates.

**Comment by qiwang97** [comment]
> > @jkhaui the one thing I liked with baseweb is the bundle size, my app is only a few Kbs in production and I love it.
> 
> Yeah, the CSS class names generation by styletron is fantastic. I much prefer it to material-ui's system which seems far too complex.
> 
> I've been building a couple of frontend apps with baseweb. Hopefully I can modify them to be generic and reusable (e.g. a dashboard). Then I'll host them somewhere as starter templates.

No better news than this today. Could you please use nextjs to start with?
